### ISSUE TITLE
Suppress noisy warnings regarding redefinition of Symbols

### ISSUE THREAD

**Issue by rod-chapman** [model-checking/cbmc-viewer#184]
CBMC-Viewer (when running in CI on the mlkem-native or mldsa-native repos) generates thousands of uninteresting warnings of the form
```
WARNING: Skipping redefinition of symbol name: MLD_65_ref_poly_add
WARNING:   Old symbol MLD_65_ref_poly_add: file mldsa/poly.c, line 52
WARNING:   New symbol MLD_65_ref_poly_add: file mldsa/poly.h, line 64
```
These appear to relate to the declaration (in the .h file) and the definition (in the .c file) of the same function, which is perfectly normal and fine.

The offending code is in:
https://github.com/model-checking/cbmc-viewer/blob/03129939546eb760150cc96495a97ef050f18355/src/cbmc_viewer/symbol_table.py#L193

These warnings make reviewing CI logs very hard.

Can these warnings be suppressed please?

**Comment by rod-chapman** [comment]
I also see a few like this:
```
WARNING: Skipping redefinition of symbol name: MLD_65_ref_invntt_tomont
WARNING:   Old symbol MLD_65_ref_invntt_tomont: file mldsa/ntt.h, line 56
WARNING:   New symbol MLD_65_ref_invntt_tomont: file MISSING, line 0
```
which I don't understand...

**Comment by rod-chapman** [comment]
I also get this warning:
```
WARNING: Skipping source file annotation: wrapped functions for code contracts
```
exactly once for every function. It adds no useful information. It is generated on line 71 of markup_code.py

Can it be suppressed please?

**Comment by rod-chapman** [comment]
I find it it effective to change these messages from `logging.warning()` to `logging.info()` so they don't appear at default verbosity level.  Is that OK?

**Comment by tautschnig** [comment]
Resolved by #187.
